docs/sql/partiql.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: default
+title: PartiQL
+parent: SQL
+nav_order: 6
+---
+
+# PartiQL
+
+SQL plugin supports PartiQL: a SQL-compatible query language that lets you query semi-structured and nested data for any data format. The SQL plugin only supports a subset of the PartiQL specification.
+To learn about PartiQL, see [PartiQL](https://partiql.org/).
```

Comment:
Remove and just link at first usage of the term, in the first sentence of the paragraph.

Reply:
Done.